docs/source/data_recipes/CDS_toolbox_template.md

```diff
@@ -0,0 +1,164 @@
+
+* CORDEX-SEA
+  
+    This data set was created with regional climate models (RCM) as part of the
+    Coordinated Regional Climate Downscaling Experiment (CORDEX). The spatial
+    resolution is 0.22 x 0.22 deg, the spatial extent is 15°S to 27°N and 89 to 146°E,
+    the temporal resolution depends on the selected period:
+  * historical data (1950-2005) is available in hourly time step
+  * scenario data (2006-2100; RCP 2.6, 4.5 and 8.5) is available in daily time step
+  
+  The full documentation and download link can be accessed [here](https://cds.climate.copernicus.eu/cdsapp#!/dataset/projections-cordex-domains-single-levels?tab=overview).
+
+* Atmospheric CO2
+  
+  Observed global CO2 levels (Mauna Loa, NOAA/GML) are available in monthly or annual
+  resolution (1958 - present) [here](https://gml.noaa.gov/ccgg/trends/graph.html).
+  Monthly data derived from satellite observation (2002 - present) is available
+  [here](https://cds.climate.copernicus.eu/cdsapp#!/dataset/satellite-carbon-dioxide?tab=overview)
+  . Alternatively, reconstructed gridded monthly CO2 data for the historical period
+  (1953 - 2013) and future CMIP6 scenarios (2015 - 2150) can be downloaded
+  [here](https://zenodo.org/record/5021361)({cite}:`cheng_wei_2021`).
+  
+## Step-by-step example
+
+Follow one of the links above to access overview information about the data set. You
+find a detailed documentation of the data set in the 'Documentation' section. To select
+data, navigate to the tab 'Download Data'.
+
+### Selection
+
+This is an example of a selection of tabs to download historical '2m air temperature'
+from the CORDEX-SEA:
+
+* Domain (South-East Asia),
+* Experiment (here: 'historical', RCPs available)
+* Horizontal resolution ('0.22 degree x 0.22 degree')
+* Temporal resolution ('daily mean')
+* Variables (here: '2m_air_temperature')
+* Global climate model (here: 'mohc_hadgem2_es')
+* Regional climate model (here: 'gerics_remo2015')
+* Ensemble member (r1i1p1)
+* Start year and End year (here: 2001-2005)
+
+Once you selected the data, you can either download the dataset for further processing,
+[see here]('./ERA5_preprocessing_example.md') an example of how to manipulate ERA5 data
+using xarray, or click on 'show Toolbox request' at the bottom of the page, copy the
+code, and open the CDS toolbox editor.
+
+### Toolbox template CORDEX-SEA
+
+The template below describes how to request a data set, reproject the data on a regular
+grid (note that the projection name is not changed!), select the area of interest,
+calculate the monthly means, and download the product. For illustration, the routine
+also plots the mean value. Adjust the 'data' lines to match your data request. You find
+the full documentation of the CDS toolbox [here](https://cds.climate.copernicus.eu/toolbox/doc/index.html).
+
+```{code-cell, eval=F, echo=T}
```

Comment:
I think this is the RMarkdown syntax for this. The simplest way to get a formatted block of code that _is not run_ is:

````md
``` ipython
````

That isn't marked as a `code-cell` so isn't run, but is a normal Markdown code formatting block. Alternatively, Myst markdown provides a richer `code-block` alternative which can have labels and titles added (https://myst.tools/docs/mystjs/code).

````md
```{code-block} ipython
````